### ISSUE TITLE
UnicodeDecodeError: 'utf-8' codec can't decode bytes in position 2336-2337: invalid continuation byte

### ISSUE THREAD

**Issue by gmolto** [grycap/scar#156]
Attempting to reproduce the scenario in examples/ffmpeg, when invoking the Lambda function (as a result of uploading a file into the S3 bucket), the following error is obtained:

```sh
UnicodeDecodeError: 'utf-8' codec can't decode bytes in position 2336-2337: invalid continuation byte
```

The partial execution log is:
```sh
START RequestId: b2bf2bb9-5b25-11e7-a290-c12043b1a386 Version: $LATEST
SCAR: Received event:
{
    "Records": [
        {
            "eventVersion": "2.0",
            "eventSource": "aws:s3",
            "awsRegion": "us-east-1",
            "eventTime": "2017-06-27T10:45:12.839Z",
            "eventName": "ObjectCreated:CompleteMultipartUpload",
            "userIdentity": {
                "principalId": "AWS:AIDAJ2QG6PUWMNOYKCICY"
            },
            "requestParameters": {
                "sourceIPAddress": "158.42.57.227"
            },
            "responseElements": {
                "x-amz-request-id": "1D5FA228CE5510A5",
                "x-amz-id-2": "V3mrbaK8gvNm3F8ZcdhqCdiw1cjlbnyi7dobtV/pVjLzz+XeA1h+N2nqrgfyI4m/ScbFIq+YKW8="
            },
            "s3": {
                "s3SchemaVersion": "1.0",
                "configurationId": "string",
                "bucket": {
                    "name": "scar-test",
                    "ownerIdentity": {
                        "principalId": "A1B93C6Q3HXTTW"
                    },
                    "arn": "arn:aws:s3:::scar-test"
                },
                "object": {
                    "key": "input/seq201.avi",
                    "size": 16745420,
                    "eTag": "d15d5c50cdfc2d3062873f958d301eae-2",
                    "sequencer": "0059523736F5B5C55A"
                }
            }
        }
    ]
}

SCAR: Container image 'sameersbn/ffmpeg' already available
SCAR: Container 'lambda_cont' already available
Downloading item from bucket scar-test with key input/seq201.avi
SCAR: Log group name: /aws/lambda/lambda-ffmpeg-00
SCAR: Log stream name: 2017/06/27/[$LATEST]d1c526d4d08b4fb2b9ebc1134ac022de
---------------------------------------------------------------------------
ERROR: Exception launched:
Traceback (most recent call last):
File "/var/task/lambda-ffmpeg-00.py", line 175, in lambda_handler
stdout += check_output(["cat", lambda_output]).decode("utf-8")
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xf2 in position 2307: invalid continuation byte
END RequestId: b2bf2bb9-5b25-11e7-a290-c12043b1a386
REPORT RequestId: b2bf2bb9-5b25-11e7-a290-c12043b1a386	Duration: 16695.57 ms	Billed Duration: 16700 ms Memory Size: 256 MB	Max Memory Used: 105 MB	
```

**Comment by gmolto** [comment]
A related error message obtained is:

```sh
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x80 in position 2302: invalid start byte
```

**Comment by gmolto** [comment]
Solved in PR #161 

Apparently, ffmpeg reads from standard input and messes the execution (see https://lists.ffmpeg.org/pipermail/ffmpeg-user/2014-May/021179.html for details). 

The solution was to replace the execution of ffmpeg from: 
```sh
ffmpeg -loglevel panic -nostats -i $SCAR_INPUT_FILE -vf format=gray $OUTPUT_FILE 
```
to

```sh
ffmpeg -loglevel panic -nostats -i $SCAR_INPUT_FILE -vf format=gray $OUTPUT_FILE  < /dev/null
```
